libcst/tests/pyre/simple_class.py

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+# fmt: off
+from typing import Sequence
+
+
+class Item:
+    def __init__(self, n: int) -> None:
+        self.number = n
+
+
+class ItemCollector:
+    def get_items(self, n: int) -> Sequence[Item]:
+        return [Item(i) for i in range(n)]
```

Comment:
The type of `range` is missing.

Reply:
Range is just a class!
https://github.com/python/typeshed/blob/master/stdlib/2and3/builtins.pyi#L1075

Pyre does seem to reveal the type of range correctly in lookup:

      "4:33-4:38/typing.Type[range]";
      "4:33-4:41/range";

Reply:
Correct, my comment was outdated, the type shows up after change to use 0.0.38.